src/core.js

```diff
@@ -425,6 +425,16 @@ jQuery.extend({
 	support: support
 });
 
+// jsHint would error on this code due to the Symbol not being defined in ES5.
+// Defining this global in .jshintrc would create a danger of using the global
+// unguarded in another place, it seems safer to just disable jsHint for these
+// three lines.
+/* jshint ignore: start */
+if ( typeof Symbol === "function" && Symbol.iterator ) {
```

Comment:
Because we don't have a reference to the global, we only have reference to `window`. In the browser it's equivalent but we also support running jQuery on Node (in a very basic sense but still) where `window` is a fake window object created by a library like jsdom.

To reference a global we'd need to pass an additional parameter to the jQuery factory... but that wouldn't work since if used with Browserify `this` references the module and not the global. So we'd basically need to run a cascade test - check for `global`, then `this`. This wouldn't be shorter and would certainly be more fragile.